ometiff-JavaScript/PixelSizeConsistency.js

```diff
@@ -1,30 +1,56 @@
-import ExifReader from '../../bids-validator/node_modules/exifreader/dist/exif-reader.js'
-import xml2js from '../../bids-validator/node_modules/xml2js/lib/xml2js.js'
+const ExifReader = require('exifreader')
+const parseString = require('xml2js').parseString
+
 
 function getExif() {
-    const tags = ExifReader.load("test_image.ome.tif");
+    const tags = ExifReader.load("test_image.ome.tif")
     return tags
 }
+
+function convertFactor(physicalSizeUnit) {
+    if(physicalSizeUnit === 'mm') {
+        return 1000
+    }else if(physicalSizeUnit === 'um') {
+        return 1
+    }else if(physicalSizeUnit === 'nm') {
+        return 0.001
+    }
```

Comment:
This is great!
OMETIFF can accept a long list of units, example [here](https://www.openmicroscopy.org/Schemas/Documentation/Generated/OME-2016-06/ome_xsd.html#Pixels_PhysicalSizeXUnit).
We don't want to take every unit possible into account, only `mm`, `um` and `nm`.

I think we should add another `else` that would lead to a warning and skip the consistency check. Something like:
"PixelSize concistency is only validated for `mm`, `um` and `nm`. PhysicalUnit in OME-TIFF is `XX`, consistency check is skipped."

Reply:
Done